lib/matplotlib/tests/test_backends_interactive.py

```diff
@@ -195,8 +195,7 @@ def _test_thread_impl():
     future = ThreadPoolExecutor().submit(fig.canvas.draw)
     plt.pause(0.5)  # flush_events fails here on at least Tkagg (bpo-41176)
     future.result()  # Joins the thread; rethrows any exception.
-    plt.close()
-    fig.canvas.flush_events()  # pause doesn't process events after close
+    plt.close()  # backend is responsible for flushing any events here
```

Comment:
It seems this needs to be reverted as it breaks wx testing.

Reply:
Am I correct in noticing that this change only produces errors in python 3.8? Maybe that should just be an XFAIL?

Reply:
I'll revert and make a TODO comment here to address it in a later PR.

Reply:
It's hard to say that, as it appears the other versions don't have wx installed. And on the other hand, macOS with Python 3.8 seems to have it, but passes.

Reply:
> It seems this needs to be reverted as it breaks wx testing.

I forgot that this change was necessary to get tkagg working after the change.  an alternative could be `plt.pause` or actually going forward with the wx xfail

Reply:
I did this.